### PR TITLE
Last features

### PR DESCRIPTION
**Pull request by Usagib**
PullRequest for StyleLint

### CODE REVIEW

**Comment by sebastianlujan** [comment]
fixed

File: css/style.css [hunk 1]
```diff
@@ -1 +1,506 @@
...
+.descr-text{
+  font-size: .8rem;
+  font-weight: bold;
+}
+
+#editors-pick .inline-display {
+  margin-bottom: 10px;
+}
+
+#editors-pick{
```
Comment (stratospherique) [inline css/style.css:498]: Don't use margins, use floats.

**Review by stratospherique — COMMENTED** [review]
## Hi Team, Great work :clap: :clap: . I like your page. however, some changes needs to be implemented in order for us to approve your project. Check blow :arrow_down: 
##### Navigation bar

* [ ] Add a 'Log in' elements on the rightmost side.

##### Footer

* [ ] It should have a fixed position (it sticks to the bottom part of the screen).


#### Implementation details

* [ ] Elements should be aligned with [float](https://css-tricks.com/all-about-floats/)
- I saw your css your are positioning elements using margins, use floats instead.
- [suggestion] use a container that contains everything but the navigation bar, and absolutely position it under the fixed nav bar. And by that, you recover the normal flow of your page :nerd_face: .

#### general notes

* [ ] The project has to have a descriptive README.md

**Comment by stratospherique** [comment]
#### That's it guys. 
**Again reopen this pull request**. fix the suggested changes. When finished, kindly resubmit another code review request **using this pull request**. Keep up the good work :muscle: :muscle: .

**Pull request closed without merging**